src/foraging_gui/Foraging.py

```diff
@@ -1522,9 +1512,8 @@ def _load_most_recent_rig_json(self,error_if_none=True):
             rig_json_path = ''
             if error_if_none:
                 logging.error('Did not find any existing rig.json files')
-                self._manage_warning_labels(self.MetadataWarning,warning_text='No rig metadata found!')
             else:
-                logging.info('Did not find any existing rig.json files')
+                logging.info('Did not find any existing rig.json files')    #FIXME: is this really the right message
```

Comment:
yes, I think thats the right error message. What are you thinking?

Reply:
When I made that comment, I didn't realize that the difference was log.info vs log.error so I was confused why the messages were the same. I will remove comment!